### PR TITLE
MS. Hatton donat. 1: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Hatton/MS_Hatton_donat_1.xml [hunk 1]
```diff
@@ -38,42 +37,565 @@
...
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">c. 1450–1460</origDate>
+                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">1450s</origDate>
                      <origPlace>
                         <country key="place_7002445">English</country> (dialect East Midland but some Northern forms)</origPlace>
                   </origin>
+                  <provenance>Sixteenth-century inscriptions: <q>William <unclear>Hurst</unclear></q> (fol. 9r), <q>Wyllm <unclear>Shirley</unclear></q> (fol. 124r), <q>Iohn Da</q> (fol. 208r).</provenance>
+                  <provenance><persName key="person_000000" role="fmo">Capt. Charles Hatton</persName>, brother of the <persName key="person_37729288">first Baron Hatton</persName> (inscribed, fol. ii recto); given to the:</provenance>
```
Comment (adunning) [inline collections/Hatton/MS_Hatton_donat_1.xml:592]: I cannot find Charles Hatton in the persons index, and his dates are not immediately obvious, though we have some other correspondence by him and I think he is <https://www.wikidata.org/wiki/Q22998036> (correspondent of Francis Bacon). NB the son of Christopher Hatton is also named Charles Hatton, <https://www.wikidata.org/wiki/Q75762697>.

File: collections/Hatton/MS_Hatton_donat_1.xml [hunk 2]
```diff
@@ -38,42 +37,565 @@
...
+                        <p>Eighteenth century, brown suade leather over pasteboard.</p>
                      </binding>
                   </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">c. 1450–1460</origDate>
+                     <origDate calendar="Gregorian" notAfter="1460" notBefore="1450">1450s</origDate>
                      <origPlace>
                         <country key="place_7002445">English</country> (dialect East Midland but some Northern forms)</origPlace>
```
Comment (holfordm) [inline collections/Hatton/MS_Hatton_donat_1.xml:589]: this conflicts with the statement above at l. 40

File: collections/Hatton/MS_Hatton_donat_1.xml (hunk 1 above)
Comment (holfordm) [reply collections/Hatton/MS_Hatton_donat_1.xml:592]: I think (?) it is the person described at pp. 255ff of http://www.northamptonshirerecordsociety.org.uk/eNpp/NppNo18_b.pdf , dates 1635-c.1708

Comment (adunning) [reply collections/Hatton/MS_Hatton_donat_1.xml]: Perfect; thank you for finding this!

**Pull request merged**